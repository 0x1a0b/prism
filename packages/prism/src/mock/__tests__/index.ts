import { Mocker } from '../index';

describe('Mocker', () => {
  let mocker: Mocker;

  beforeEach(() => {
    mocker = new Mocker();
  });

  describe('mock()', () => {
    it('mocks', () => {
      expect(mocker.mock('test')).toMatchSnapshot();
    });
  });
});
