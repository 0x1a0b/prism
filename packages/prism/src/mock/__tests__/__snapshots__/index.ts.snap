// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mocker mock() mocks 1`] = `Promise {}`;
